lib/engine/game/g_1880/step/choose.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class Choose < Engine::Step::Base
+          ACTIONS = %w[choose pass].freeze
+
+          def actions(entity)
+            return [] unless entity == current_entity
+            return [] if !find_company(entity) || @round.already_passed
+
+            ACTIONS
+          end
+
+          def auto_actions(entity)
+            return [Engine::Action::Pass.new(entity)] if !find_company(entity) || @round.already_passed
+
+            return [Engine::Action::Choose.new(entity, choice: '')] if @game.phase.name == 'B2'
+
+            []
+          end
+
+          def choice_name
+            return "Claim #{@company.name} Award" if @company
+
+            'Choose'
+          end
+
+          def choices
+            return ["Claim Award (#{@game.format_currency(sell_price)})"] if @company
+
+            {}
+          end
+
+          def round_state
+            super.merge(
+            { already_passed: false }
+          )
+          end
+
+          def description
+            'Claim Award'
+          end
+
+          def process_choose(action)
+            action.entity.companies.delete(@company)
+            @company.close!
+            @game.bank.spend(sell_price, current_entity) if sell_price.positive?
+            @log << "#{action.entity.name} claims #{@game.format_currency(sell_price)} award for #{@company.name}"
+            @company = nil
+            pass!
+          end
+
+          def hide_corporations?
+            true
+          end
+
+          def choice_available?(_entity)
+            true
+          end
+
+          def skip!
+            pass!
+          end
+
+          def can_sell?
```

Comment:
Nit: It's not really a sell, but a one-time payment

Reply:
moved from when this was a sell_company action. renamed